pkg/collector/corechecks/cluster/orchestrator/collector_bundle.go

```diff
@@ -0,0 +1,145 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubeapiserver && orchestrator
```

Comment:
Wondering if we should keep these or not? It seems they are not added yet because of the go version used to release the agent

Reply:
I think go 1.16 accepts both syntax so IMO it's safer to use both for the sake of a smooth transition later on.

Reply:
yes there is currently a PR which does that for all files. https://github.com/DataDog/datadog-agent/pull/10601